src/index.ts

```diff
@@ -17,7 +17,10 @@ import {
   ObfuscatedFlag,
   BoundedEventQueue,
   validation,
+  PrecomputedFlag,
+  NamedEventQueue,
 } from '@eppo/js-client-sdk-common';
+import { Event } from '@eppo/js-client-sdk-common/src/events/event-dispatcher';
```

Comment:
@felipecsl I needed to do an `as NamedEventQueue<Event>` in order get an event type to work with `newDefaultEventDispatcher`. Am I doing this right? Are you okay with including `Event` in the public exports?

Reply:
ah yeah we should be exporting that! i'll send a PR!

Reply:
https://github.com/Eppo-exp/js-sdk-common/pull/171